StripeConnect/StripeConnectTests/WebView+Tests.swift

```diff
@@ -135,23 +137,23 @@ extension WKWebView {
     @discardableResult
     func evaluateMessageWithReply(name: String,
```

Comment:
optional:
I didn't catch this in the previous PR where this helper was added – given that this method is already async, does `verifyResult` need to be a closure? Instead, we could directly return the result and then assert on the result value in the test. This would eliminate the need for an expectation.

So this could look something like:

```
@MainActor
func testSomething() async {
    let result = await webView.evaluateMessageWithReply(name: "foo", json: "{}")
    XCTAssertEquals(result, ...)
}
```